metadata-db-app/int_test/cmr/metadata_db/int_test/concepts/force_delete_test.clj

```diff
@@ -1,19 +1,28 @@
 (ns cmr.metadata-db.int-test.concepts.force-delete-test
   "Contains integration test for emptying database via force-delete."
-  (:require [clojure.test :refer :all]
-            [clj-http.client :as client]
-            [cheshire.core :as cheshire]
-            [cmr.metadata-db.int-test.utility :as util]
-            [cmr.metadata-db.int-test.concepts.concept-save-spec :as c-spec]
-            [cmr.metadata-db.int-test.concepts.concept-delete-spec :as cd-spec]))
+  (:require
+   [cheshire.core :as cheshire]
+   [clj-http.client :as client]
+   [clojure.test :refer :all]
+   [cmr.metadata-db.int-test.concepts.concept-delete-spec :as cd-spec]
+   [cmr.metadata-db.int-test.concepts.concept-save-spec :as cs-spec]
+   [cmr.metadata-db.int-test.utility :as util]))
 
-;;; fixtures
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-(use-fixtures :each (util/reset-database-fixture {:provider-id "REG_PROV" :small false}
-                                                 {:provider-id "SMAL_PROV" :small true}))
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;; Fixtures & one-off utility functions
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
-;;; tests
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+(use-fixtures :each (util/reset-database-fixture
+                     {:provider-id "REG_PROV" :small false}
+                     {:provider-id "SMAL_PROV" :small false}))
+
+(defmethod cs-spec/gen-concept :service
```

Comment:
Not sure how I feel about that ... if this bit of code is important to reuse elsewhere, it should go in utility; if, on the other hand, it's too specific a method for that, it might just be best to leave it like it is ...

I'll leave it for now, until we think of a good solution.

Reply:
That's fine.